bundles/org.openhab.binding.amazondashbutton/src/main/java/org/openhab/binding/amazondashbutton/internal/discovery/AmazonDashButtonDiscoveryService.java

```diff
@@ -212,7 +212,7 @@ public void packetCaptured(MacAddress macAddress) {
                     // @formatter:off
                     DiscoveryResult discoveryResult = DiscoveryResultBuilder.create(dashButtonThing)
                             .withLabel("Dash Button")
-                            .withRepresentationProperty(macAdressString)
```

Comment:
Just a minor nit-picking comment - while you're at it anyways: `PROPERTY_MAC_ADDRESS` now is part of the [Thing API](https://github.com/openhab/openhab-core/blob/46666eb169d8f1b02cf7a2f3f5b54bc83c0784ed/bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/Thing.java#L57-L58) and can be use globally. 😉